Sming/Libraries/MPU6050/README.rst

```diff
@@ -0,0 +1,14 @@
+API Documentation
+-----------------
+
+.. doxygenclass:: MPU6050
+   :members:
+
```

Comment:
Move this to the bottom of the file for consistency - it gets expanded into a lot of information.